src/main/java/reposense/report/ReportGenerator.java

```diff
@@ -348,8 +382,10 @@ private static AnalyzeJobOutput analyzeRepos(String outputPath, List<RepoConfigu
     }
 
     /**
-     * Analyzes repo specified by {@code config} and generates the report.
+     * Analyzes repo specified by {@code config} and generates the report at {@code repoReportDirectory}.
+     *
      * @return A list of paths to the JSON report files generated for the repo specified by {@code config}.
+     * @throws NoAuthorsWithCommitsFoundException if there are no authors with commits found for the repo.
      */
     private static List<Path> analyzeRepo(
             RepoConfiguration config, String repoReportDirectory) throws NoAuthorsWithCommitsFoundException {
```

Comment:
Should move the `config` and `repoReportDirectory` to line 390 and keep only the `throws` portion in line 391.